packages/webview_flutter/webview_flutter/lib/src/webview.dart

```diff
@@ -90,10 +90,12 @@ class WebView extends StatefulWidget {
     this.debuggingEnabled = false,
     this.gestureNavigationEnabled = false,
     this.userAgent,
+    this.zoomEnabled = true,
     this.initialMediaPlaybackPolicy =
         AutoMediaPlaybackPolicy.require_user_action_for_all_media_types,
     this.allowsInlineMediaPlayback = false,
   })  : assert(javascriptMode != null),
+        assert(zoomEnabled != null),
```

Comment:
nit: Since `zoomEnabled` is non-null, this shouldn't need to be null checked

Reply:
NNBD FTW! Fixed in f13a0dd